### PR TITLE
Delete service docs

### LINKED ISSUE #77: Allow deletion of service-doc in UI
* Potentially in a separate management tab or introduce a "danger zone" in each service detail view to delete
* API path: `DELETE /service-docs/{service-name}`

### PR DESCRIPTION
**Pull request by MBuchalik**
> **Warning**
> This PR is based on #117, so please merge #117 first. For a better diff see https://github.com/osrgroup/msadoc/compare/1490130809044895dd72712d4f54e80b6589036f...MBuchalik:77-delete-service-docs

Closes #77: There is now a "Danger Zone" that allows users to delete a Service Doc.

Also, I noticed that in the tree on the left, the services and groups would not get sorted alphabetially. This is now also fixed.

# Screenshots

grafik
grafik
grafik
grafik
grafik

### CODE REVIEW

File: frontend/src/services/http/service-docs.tsx [hunk 1]
```diff
@@ -57,12 +62,49 @@ function useServiceDocsHttpService(): ServiceDocsHttpService {
...
+        data: response.data,
+      };
+    }
+    return {
+      status: 0,
+      data: undefined,
+    };
+  }
+
+  async function doDeleteSingleApiKey(
```
Comment (georg-schwarz) [inline frontend/src/services/http/service-docs.tsx:85]: `doDeleteSingleServiceDoc`?

**Comment by georg-schwarz** [comment]
Looks like I stirred up a NPM issue with the registries... Gonna look into it later.

File: frontend/src/services/http/service-docs.tsx (hunk 1 above)
Comment (MBuchalik) [reply frontend/src/services/http/service-docs.tsx:85]: Good catch! I did a bit of copy&pasting between the two services. Looks like I missed renaming it 😄

Comment (MBuchalik) [reply frontend/src/services/http/service-docs.tsx]: Done 👍

**Comment by MBuchalik** [comment]
I have fixed the wrong function name. Can we merge this?

**Pull request merged**